### PR TITLE
change color variables

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/accounting/ng-virtual-list/virtual-list.component.css [hunk 1]
```diff
@@ -95,7 +95,7 @@ i.close.icon.clear:hover::after {
 
 .sh-select-disabled {
     background-color: #e3e3e3;
-    color: darkgray;
+    color: $bay-Of-many;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/accounting/ng-virtual-list/virtual-list.component.css:98]: please don't use capital letters

File: apps/web-giddh/src/app/contact/aging-report/aging-report.component.scss [hunk 2]
```diff
@@ -35,7 +35,7 @@
     width: auto;
 }
 .wrapsearchingAging a {
-    color: $JacksonsPurple;
+    color: $Jacksons-purple;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/aging-report/aging-report.component.scss:38]: please don't use capital letters

File: apps/web-giddh/src/app/contact/aging-report/aging-report.component.scss [hunk 3]
```diff
@@ -61,19 +61,19 @@
...
     padding: 4px 9px;
-    background-color: $border-blue;
+    background-color: $chetwode-blue;
     color: #fff;
     font-size: 20px;
     border-radius: 2px;
 }
 tr.aging-table-top {
-    border-bottom: 1px solid $tableBorderColor;
+    border-bottom: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/aging-report/aging-report.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 4]
```diff
@@ -139,7 +139,7 @@ ul.list-unstyled li a {
     }
 
     a:hover {
-        color: $JacksonsPurple !important;
+        color: $Jacksons-purple !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss:142]: please don't use capital letters

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 5]
```diff
@@ -243,7 +243,7 @@ tr.aging-table-top td {
 
 .moreBtn li:hover {
     background-color: #eeeff9;
-    color: $JacksonsPurple;
+    color: $Jacksons-purple;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss:246]: please don't use capital letters

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 6]
```diff
@@ -262,7 +262,7 @@ tr.aging-table-top td {
 i.icon-options {
     width: 36px;
     height: 36px;
-    border: 1px solid $inputBorderColor;
+    border: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 7]
```diff
@@ -303,8 +303,8 @@ span.inputAddon {
     min-width: 140px;
 }
 .aging-table thead tr th {
-    border-top: 1px solid $tableBorderColor;
-    border-bottom: 1px solid $tableBorderColor !important;
+    border-top: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 8]
```diff
@@ -303,8 +303,8 @@ span.inputAddon {
...
+    border-bottom: 1px solid $Gainsboro !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.component.scss [hunk 9]
```diff
@@ -287,9 +287,9 @@ tfoot td .tableSec.form-group {
     align-items: center;
     font-size: 18px;
     margin-left: 10px;
-    border: 1px solid $inputBorderColor;
+    border: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/inventory/components/sidebar-components/stockList.component.scss [hunk 10]
```diff
@@ -4,7 +4,7 @@
     background-color: #eeeff9;
 }
 .stock-items .active a .span {
-    color: $JacksonsPurple !important;
+    color: $Jacksons-purple !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/components/sidebar-components/stockList.component.scss:7]: please don't use capital letters

File: apps/web-giddh/src/app/inventory/components/sidebar-components/stockList.component.scss [hunk 11]
```diff
@@ -18,5 +18,5 @@
     line-height: 14px !important;
 }
 .stock-items .active .btn-link .fa {
-    color: $JacksonsPurple;
+    color: $Jacksons-purple;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/components/sidebar-components/stockList.component.scss:21]: please don't use capital letters

File: apps/web-giddh/src/app/inventory/components/sidebar-components/stockgrplist.component.scss [hunk 12]
```diff
@@ -1,6 +1,6 @@
 @import "../../../../assets/css/color.scss";
 .active {
-    color: $JacksonsPurple !important;
+    color: $Jacksons-purple !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/components/sidebar-components/stockgrplist.component.scss:3]: please don't use capital letters

File: apps/web-giddh/src/app/inventory/components/sidebar-components/stockgrplist.component.scss [hunk 13]
```diff
@@ -22,7 +22,7 @@
 .stock-grp-list .active a,
 s.tock-grp-list .active i,
 .stock-grp-list .active .btn-link i {
-    color: $JacksonsPurple !important;
+    color: $Jacksons-purple !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/components/sidebar-components/stockgrplist.component.scss:25]: please don't use capital letters

File: apps/web-giddh/src/app/invoice/create/invoice.create.component.scss [hunk 14]
```diff
@@ -23,7 +23,7 @@
     width: 100%;
     height: 80px;
     padding: 20px 0;
-    background-color: "$veryLightGray";
+    background-color: "$very-light-gray";
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/create/invoice.create.component.scss:26]: why are we using double quotes here?

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 15]
```diff
@@ -309,7 +309,7 @@ ul.selectActionDropdown {
     width: 100%;
 }
 .input-group input.form-control:focus {
-    border-color: $inputBorderColor;
+    border-color: $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 16]
```diff
@@ -319,7 +319,7 @@ ul.selectActionDropdown {
     display: inline-flex;
     justify-content: center;
     align-items: center;
-    border-right: 1px solid $inputBorderColor;
+    border-right: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 17]
```diff
@@ -331,31 +331,31 @@ ul.selectActionDropdown {
     align-items: center;
     font-size: 18px;
     margin-left: 10px;
-    border: 1px solid $inputBorderColor;
+    border: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss [hunk 18]
```diff
@@ -192,7 +192,7 @@ span.eligible-ITC {
 }
 .adjust-receipt {
     padding: 10px;
-    border: 1px solid $inputBorderColor;
+    border: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss [hunk 19]
```diff
@@ -211,7 +211,7 @@ span.eligible-ITC {
     display: flex;
 }
 .ledger-head th {
-    color: $JacksonsPurple;
+    color: $Jacksons-purple;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss:214]: please don't use capital letters

File: apps/web-giddh/src/app/ledger/ledger.component.scss [hunk 20]
```diff
@@ -104,7 +104,7 @@ input {
     bottom: 0;
 }
 .ledger-head th {
-    color: $JacksonsPurple;
+    color: $Jacksons-purple;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/ledger.component.scss:107]: please don't use capital letters

File: apps/web-giddh/src/app/ledger/ledger.component.scss [hunk 21]
```diff
@@ -388,17 +388,17 @@ sh-select:focus {
     padding-bottom: 20px;
 }
 .stat-block:first-child {
-    border-bottom: 8px solid $dark-blue00;
+    border-bottom: 8px solid $midnight-express;
 }
 .stat-block:nth-child(2) {
-    border-bottom: 8px solid $JacksonsPurple;
+    border-bottom: 8px solid $Jacksons-purple;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/ledger.component.scss:394]: please don't use capital letters

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.scss [hunk 22]
```diff
@@ -33,7 +33,7 @@
         cursor: default;
         width: 100%;
         &:focus {
-            border-color: $inputBorderColor;
+            border-color: $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.scss [hunk 23]
```diff
@@ -45,7 +45,7 @@
     display: inline-flex;
     justify-content: center;
     align-items: center;
-    border-right: 1px solid $inputBorderColor;
+    border-right: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss [hunk 24]
```diff
@@ -17,7 +17,7 @@
 }
 
 .branch-transfer-report .basic > thead > tr > th {
-    border-top: 1px solid $tableBorderColor !important;
+    border-top: 1px solid $Gainsboro !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss [hunk 25]
```diff
@@ -37,7 +37,7 @@
     display: inline-flex;
     justify-content: center;
     align-items: center;
-    border-right: 1px solid $tableBorderColor;
+    border-right: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss [hunk 26]
```diff
@@ -46,14 +46,14 @@
     padding-left: 45px !important;
     width: 210px;
     &:focus {
-        border-color: $tableBorderColor !important;
+        border-color: $Gainsboro !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 27]
```diff
@@ -23,7 +23,7 @@
 }
 
 .account-search .input-group-addon {
-    border-color: $inputBorderColor;
+    border-color: $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 28]
```diff
@@ -41,15 +41,15 @@
 
 .account-search .form-control {
     background: none !important;
-    border-color: $inputBorderColor;
+    border-color: $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 29]
```diff
@@ -41,15 +41,15 @@
...
     border-left: 0px;
     border-radius: 0px 2px 2px 0 !important;
     height: 36px;
     padding-left: 3px !important;
 }
 
 .account-search .form-control:focus {
-    border-color: $inputBorderColor !important;
+    border-color: $Gainsboro !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.scss]: please don't use capital letters

File: apps/web-giddh/src/app/theme/account-detail-modal/account-detail-modal.component.scss [hunk 30]
```diff
@@ -79,7 +79,7 @@ ul.list-unstyled li a {
 .li-link {
     margin-top: 15px !important;
     a:hover {
-        color: $JacksonsPurple !important;
+        color: $Jacksons-purple !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/account-detail-modal/account-detail-modal.component.scss:82]: please don't use capital letters

File: apps/web-giddh/src/app/theme/ng-virtual-select/sh-select.component.css [hunk 31]
```diff
@@ -95,7 +95,7 @@ i.close.icon.clear:hover::after {
 
 .sh-select-disabled {
     background-color: #e3e3e3;
-    color: darkgray;
+    color: $bay-Of-many;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/ng-virtual-select/sh-select.component.css:98]: please don't use capital letters

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss [hunk 32]
```diff
@@ -97,7 +98,7 @@ i.close.icon.clear:hover::after {
 
 .sh-select-disabled {
     background-color: #e3e3e3;
-    color: darkgray;
+    color: $bay-Of-many;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:101]: please don't use capital letters

File: apps/web-giddh/src/assets/css/table.scss [hunk 33]
```diff
@@ -1,7 +1,7 @@
 .giddh-table {
     background: #fff;
     font-family: $tableFontSemibold;
-    border: 1px solid $tableBorderColor;
+    border: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/table.scss]: please don't use capital letters

File: apps/web-giddh/src/assets/css/table.scss [hunk 34]
```diff
@@ -16,11 +16,11 @@
         border-right: none;
         background: #fff;
         padding: 15px 11px;
-        border-bottom: 1px solid $tableBorderColor;
+        border-bottom: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/table.scss]: please don't use capital letters

File: apps/web-giddh/src/assets/css/table.scss [hunk 35]
```diff
@@ -16,11 +16,11 @@
...
         font-family: $tableFontSemibold;
     }
     thead > tr > th {
-        border-top: 1px solid $tableBorderColor !important;
+        border-top: 1px solid $Gainsboro !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/table.scss]: please don't use capital letters

File: apps/web-giddh/src/assets/css/table.scss [hunk 36]
```diff
@@ -39,13 +39,13 @@
     }
     thead > tr > th:first-child,
     tbody > tr > td:first-child {
-        border-left: 1px solid $tableBorderColor;
+        border-left: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/table.scss]: please don't use capital letters

File: apps/web-giddh/src/assets/css/table.scss [hunk 37]
```diff
@@ -39,13 +39,13 @@
...
         white-space: normal !important;
         word-break: inherit;
     }
     thead > tr > th:last-child,
     tbody > tr > td:last-child {
-        border-right: 1px solid $tableBorderColor;
+        border-right: 1px solid $Gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/table.scss]: please don't use capital letters

File: apps/web-giddh/src/assets/css/table.scss [hunk 38]
```diff
@@ -57,14 +57,14 @@
         background: rgba(221, 225, 230, 0.5);
     }
     tbody tr:last-child td {
-        border-bottom: 1px solid $tableBorderColor !important;
+        border-bottom: 1px solid $Gainsboro !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/table.scss]: please don't use capital letters

File: apps/web-giddh/src/app/daybook/daybook.component.scss [hunk 39]
```diff
@@ -1,6 +1,6 @@
 .date-range-picker {
     padding-left: 35px !important;
-    // border-color:$border-blue;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/daybook/daybook.component.scss:3]: Remove commented css

File: apps/web-giddh/src/app/daybook/daybook.component.scss [hunk 40]
```diff
@@ -1,6 +1,6 @@
...
+    // border-color:$chetwode-blue;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/daybook/daybook.component.scss:3]: Remove commented css

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.scss [hunk 41]
```diff
@@ -491,15 +491,15 @@ dropdown-menu-right {
...
-    color: $light-text-blue;
+    color: $chetwode-blue;
     border: 0px;
     border-radius: 4px;
-    border: 1px solid $light-text-blue;
+    border: 1px solid $chetwode-blue;
 }
 
 .btn-2:hover {
     color: #ffffff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.scss:501]: use colour var

Comment (sadik-walkover) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.scss:501]: use color var

File: apps/web-giddh/src/app/accounting/ng-virtual-list/virtual-list.component.css (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/accounting/ng-virtual-list/virtual-list.component.css]: done

File: apps/web-giddh/src/app/contact/aging-report/aging-report.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/contact/aging-report/aging-report.component.scss]: done

File: apps/web-giddh/src/app/contact/aging-report/aging-report.component.scss (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/contact/aging-report/aging-report.component.scss]: done

File: apps/web-giddh/src/app/contact/contact.component.scss (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/contact/contact.component.scss]: done

File: apps/web-giddh/src/app/contact/contact.component.scss (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/contact/contact.component.scss]: done

File: apps/web-giddh/src/assets/css/table.scss (hunk 36 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/table.scss]: done

File: apps/web-giddh/src/assets/css/table.scss (hunk 37 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/table.scss]: done

File: apps/web-giddh/src/assets/css/table.scss (hunk 38 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/table.scss]: done

File: apps/web-giddh/src/app/inventory/components/sidebar-components/stockgrplist.component.scss (hunk 12 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/inventory/components/sidebar-components/stockgrplist.component.scss]: done

File: apps/web-giddh/src/app/inventory/components/sidebar-components/stockgrplist.component.scss (hunk 13 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/inventory/components/sidebar-components/stockgrplist.component.scss]: done

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss (hunk 15 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss]: done

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss (hunk 16 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss]: done

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss (hunk 18 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss]: done

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss (hunk 17 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss]: done

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss (hunk 19 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss]: done

File: apps/web-giddh/src/app/ledger/ledger.component.scss (hunk 20 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.scss]: done

File: apps/web-giddh/src/app/ledger/ledger.component.scss (hunk 21 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.scss]: done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.scss (hunk 22 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.scss]: done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.scss (hunk 23 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.scss]: done

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss (hunk 24 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss]: done

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss (hunk 25 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss]: done

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss (hunk 26 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.scss]: done

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 28 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.scss]: done

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 29 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.scss]: done

File: apps/web-giddh/src/app/theme/ng-virtual-select/sh-select.component.css (hunk 31 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/theme/ng-virtual-select/sh-select.component.css]: done

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss (hunk 32 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss]: done

File: apps/web-giddh/src/assets/css/table.scss (hunk 33 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/table.scss]: done

File: apps/web-giddh/src/app/contact/contact.component.scss (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/contact/contact.component.scss]: done

File: apps/web-giddh/src/app/inventory/components/sidebar-components/stockList.component.scss (hunk 11 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/inventory/components/sidebar-components/stockList.component.scss]: done

File: apps/web-giddh/src/app/invoice/create/invoice.create.component.scss (hunk 14 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/create/invoice.create.component.scss:26]: i am not using here , already added

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 27 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.scss]: done

File: apps/web-giddh/src/app/theme/account-detail-modal/account-detail-modal.component.scss (hunk 30 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/theme/account-detail-modal/account-detail-modal.component.scss]: done

File: apps/web-giddh/src/app/invoice/create/invoice.create.component.scss (hunk 14 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/create/invoice.create.component.scss]: Please remove double quotes from here even if you have not added as it is incorrect

File: apps/web-giddh/src/app/login/login.component.scss [hunk 42]
```diff
@@ -474,7 +474,7 @@ small {
     overflow: hidden;
 }
 .logo-login {
-    background-color: $darkMidnightBlue;
+    background-color: $darkmid-night-blue;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/login/login.component.scss:477]: Should be `dark-midnight-blue`

File: apps/web-giddh/src/app/login/login.component.scss [hunk 43]
```diff
@@ -484,46 +484,46 @@ small {
 .accounting-space {
     padding: 30px;
     border-radius: 0;
-    background-color: $White;
+    background-color: $white;
 }
 .accounting-space h4 {
-    color: $pageTitleColor;
+    color: $mid-night-blue;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/login/login.component.scss:490]: Should be `midnight-blue`

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss [hunk 44]
```diff
@@ -17,7 +17,7 @@
 }
 .date-range-picker {
     padding-left: 35px !important;
-    // border-color:$border-blue;
+    // border-color:$chetwode-blue;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss:20]: Do remove commented styles

File: apps/web-giddh/src/app/theme/ng-virtual-select/sh-select-menu.component.scss [hunk 45]
```diff
@@ -26,17 +26,17 @@
 }
 
 // .item:hover {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/theme/ng-virtual-select/sh-select-menu.component.scss:28]: Do remove commented styles

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 46]
```diff
@@ -3693,7 +3693,7 @@ a.btn.btn-xs.btn-link.pull-right {
 
 /* #shareModalh3 {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:3694]: Do remove commented styles

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 47]
```diff
@@ -3601,15 +3601,15 @@ a.btn.btn-xs.btn-link.pull-right {
 }
 
 /* .form-control[readonly] {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:3603]: Do remove commented styles

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 48]
```diff
@@ -5586,7 +5586,7 @@ input[type="checkbox"][disabled] {
 }
 
 .ledger-row .select2-ledger .select2-container--open .select2-search__field {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:5588]: Do remove commented styles

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 49]
```diff
@@ -5890,13 +5890,13 @@ ng-select > div > div.single > div.clear:hover {
 }
 
 .input-group .input-group-btn .btn {
-    /*background: $White;*/
+    /*background: $white;*/
     border-left: 0px;
 }
 
 // .input-group .input-group-btn .btn:hover {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:5897]: Do remove commented styles

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 50]
```diff
@@ -5890,13 +5890,13 @@ ng-select > div > div.single > div.clear:hover {
 }
 
 .input-group .input-group-btn .btn {
-    /*background: $White;*/
+    /*background: $white;*/
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:5893]: Do remove commented styles

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 51]
```diff
@@ -8313,13 +8313,13 @@ ul.dropdown-menu p.lead.pd {
 }
 
 /*  .nav.nav-tabs>li>a {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:8315]: Do remove commented styles

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 47 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 48 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 50 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 51 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: done

File: apps/web-giddh/src/app/login/login.component.scss (hunk 43 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/login/login.component.scss]: done

**Pull request merged**